src/base/config.lua

```diff
@@ -49,10 +49,14 @@
 		local bundlepath = ""
 
 		if cfg.system == p.MACOSX and kind == p.WINDOWEDAPP then
```

Comment:
Could this also apply to `p.iOS`?

Reply:
I did not specifically think about `system: ios`.
Probably it is better to apply.